testing/constraints-3.8.txt

```diff
@@ -1 +1,12 @@
+# This constraints file is used to check that lower bounds
+# are correct in setup.py
+# List *all* library dependencies and extras in this file.
+# Pin the version to the lower bound.
+#
+# e.g., if setup.py has "foo >= 1.14.0, < 2.0.0dev",
 sqlalchemy==1.3.24
```

Comment:
Need to either change this to 1.2.0 or update the minimum version in `setup.py`.